src/LearningObjects/LearningObjectRouteHandler.spec.ts

```diff
@@ -47,16 +47,52 @@ describe('LearningObjectRouteHandler', () => {
                 .get(`/learning-objects/${testObjectID}`)
                 .expect(200)
                 .then(res => {
-                    expect(res.text).toContain(`${testObjectID}`);
+                    expect(JSON.parse(res.text)).toMatchObject({
+                        author: {
+                             bio: expect.any(String),
+                             createdAt: expect.any(String),
+                             email: expect.any(String),
+                             emailVerified: true,
+                             id: expect.any(String),
+                             name: expect.any(String),
+                             organization: expect.any(String),
+                             username: expect.any(String)},
+                        children: expect.any(Array),
+                        collection: expect.any(String),
+                        contributors: expect.any(Array),
+                        date: expect.any(String),
+                        description: expect.any(String),
+                        id: testObjectID,
+                        length: expect.any(String),
+                        levels: expect.arrayContaining([expect.any(String)]),
+                        materials: {
+                             files: expect.any(Array),
+                             folderDescriptions: expect.any(Array),
+                             notes: expect.any(String),
+                             pdf: expect.objectContaining(
+                                 {name: expect.any(String),
+                                  url: expect.any(String)}),
+                             urls: expect.arrayContaining(
+                                 [expect.objectContaining(
+                                 {title: expect.any(String),
+                                  url: expect.any(String)}),
+                                 ]),
+                    },
+                        metrics: {
+                             downloads: expect.any(Number),
+                             saves: expect.any(Number)},
+                        name: 'Learn Unit Testing',
```

Comment:
Keep test generic like other cases.
```suggestion
                        name: expect.any(String),
```